docs/src/developer-guide/working-with-rules.md

```diff
@@ -346,18 +346,19 @@ Best practices for fixes:
 1. Make fixes as small as possible. Fixes that are unnecessarily large could conflict with other fixes, and prevent them from being applied.
 1. Only make one fix per message. This is enforced because you must return the result of the fixer operation from `fix()`.
 1. Since all rules are run again after the initial round of fixes is applied, it's not necessary for a rule to check whether the code style of a fix will cause errors to be reported by another rule.
+
     * For example, suppose a fixer would like to surround an object key with quotes, but it's not sure whether the user would prefer single or double quotes.
 
         ```js
-        ({ foo : 1 })
-
-        // should get fixed to either
+        ({ foo: 1 }(
+            // should get fixed to either
 
-        ({ 'foo': 1 })
+            { foo: 1 }
+        )(
+            // or
 
-        // or
-
-        ({ "foo": 1 })
+            { foo: 1 }
+        ));
         ```
```

Comment:
This example looks broken now. What do you think about reverting the formatting changes, as they created a huge diff that is unrelated to this fix? I'm also not sure if this is formatting we would like to enforce, like 3 spaces after `*`, trailing commas in js code blocks, etc.

Reply:
Yeah, I will revert this formatting change. This got automatically formatted maybe I need to update my vscode extension settings.